vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/batchRegistration/SampleRegistrationDialog.java

```diff
@@ -23,17 +23,16 @@
  */
 public class SampleRegistrationDialog extends Dialog {
 
-  private final Span title = new Span("Upload Laboratory Metadata");
+  private final String TITLE = "Register Batch";
   private final TabSheet tabStepper = new TabSheet();
-  private final Tab generalInformationTab = createTabStep("1", "General Information");
-  private final Tab sampleMetadataTab = createTabStep("2", "Sample Metadata");
+  private final Tab generalInformationTab = createTabStep("1", "Batch Information");
+  private final Tab sampleMetadataTab = createTabStep("2", "Register Samples");
```

Comment:
We might want to rename the variables as well.

Reply:
done!